src/CalcViewModel/Common/CopyPasteManager.cpp

```diff
@@ -449,33 +448,37 @@ bool CopyPasteManager::TryOperandToULL(const wstring& operand, int numberBase, u
     {
         // Do nothing
     }
+    catch(...)
+    {
+        // Do nothing
+    }
 
     return false;
 }
 
 size_t CopyPasteManager::OperandLength(wstring operand, ViewMode mode, CategoryGroupType modeType, int programmerNumberBase)
 {
-    size_t len = 0;
     if (mode == ViewMode::Standard || mode == ViewMode::Scientific)
     {
-        len = StandardScientificOperandLength(operand);
+        return StandardScientificOperandLength(operand);
     }
     else if (mode == ViewMode::Programmer)
```

Comment:
We should probably use a `switch` to improve the readability of the code

Reply:
Latest commit on this branch accomplishes this.